lms/djangoapps/course_api/blocks/tests/test_api.py

```diff
@@ -165,5 +165,5 @@ def test_query_counts_uncached(self, store_type_tuple, with_storage_backing):
             self._get_blocks(
                 course,
                 expected_mongo_queries,
-                expected_sql_queries=9 if with_storage_backing else 3,
+                expected_sql_queries=11 if with_storage_backing else 3,
```

Comment:
**Note to reviewers:** This query count went up by 2 because of the new `transaction.atomic()` call later in this PR (introducing `Savepoint` and `Release Savepoint` SQL calls).